application/src/main/java/org/thingsboard/server/service/queue/DefaultTbClusterService.java

```diff
@@ -167,7 +165,10 @@ public void pushMsgToRuleEngine(TopicPartitionInfo tpi, UUID msgId, ToRuleEngine
 
     @Override
     public void pushMsgToRuleEngine(TenantId tenantId, EntityId entityId, TbMsg tbMsg, TbQueueCallback callback) {
-        if (tenantId.isNullUid()) {
+        if (tenantId == null) {
```

Comment:
Wrong fix. Make the same change. if (tenantId == null || tenantId.isNullUid()) {